### PR TITLE
fix trigger time recording in single_template, also whitespace (alas)

### PR DESCRIPTION
**Pull request by tdent**
in addition to fixing the bug of not always recording the input trigger time value, disallow inconsistent options.

### CODE REVIEW

**Comment by tdent** [comment]
Has been minimally tested not to break the executable when --window and --trigger-time are specified.

File: bin/pycbc_single_template [hunk 1]
```diff
@@ -85,19 +85,22 @@ def select_segments(fname, anal_name, data_name, ifo, time, pad_data):
...
                   help="The low frequency cutoff to use for filtering (Hz)")
 parser.add_argument("--chisq-bins", default="0", type=str, help=
                     "Number of frequency bins to use for power chisq.")
 parser.add_argument("--minimum-chisq-bins", default=0, type=int, help=
                     "If the chisq bin formula fails, default to this number.")
-parser.add_argument("--trigger-time", type=float, default=0,
-                  help="The central time of the trigger to use.")
+# flag for trigger time not being entered
+stupid_time = -999999999.
+parser.add_argument("--trigger-time", type=float, default=stupid_time,
```
Comment (spxiwh) [inline bin/pycbc_single_template]: Why do you need stupid_time? Could you not use:
```
default=None
```
and then check for:
```
if opt.trigger_time is [not] None
```
below

Comment (tdent) [reply bin/pycbc_single_template]: I suppose so.

Comment (tdent) [reply bin/pycbc_single_template]: See the revised diff.

**Pull request merged**